.travis.yml

```diff
@@ -1,8 +1,31 @@
-language: objective-c
-
```

Comment:
Should we set the language to `node_js` so it doesn't load up the Ruby container?